jellyfin_mpv_shim/utils.py

```diff
@@ -230,10 +231,18 @@ def get_resource(*path):
     # Detect if bundled via pyinstaller.
     # From: https://stackoverflow.com/questions/404744/
     if getattr(sys, "_MEIPASS", False):
-        application_path = os.path.join(getattr(sys, "_MEIPASS"), "jellyfin_mpv_shim")
+        if platform.system() == "Darwin":
```

Comment:
Is `_MEIPASS` set here? That should only be with pyinstaller and it appears you are doing a check below looking to see if the application is frozen.

Reply:
Yeah I forgot to remove that check because adding it to _MIEPASS worked for me.